<launch>
    <arg name="my_name" default="dory" />
    <arg name="blue_team_names" default="[dory, guppy]" /> <!-- Includes this agent -->
    <arg name="blue_team_positions_enu" default="[[0.01,0.01,0.01],[]]" /> <!-- Includes this agent, can be blank for unknown starting position -->

    <arg name="topside_name" default="topside" />
    <arg name="red_team_name" default="" />
    <arg name="config_file" default="$(find etddf_minau)/config/etddf.yaml" />

    <node name="etddf" pkg="etddf_minau" type="etddf_node_new.py" output="screen">
        <param name="my_name" value="$(arg my_name)" />
        <rosparam param="blue_team_positions" subst_value="True">$(arg blue_team_positions_enu)</rosparam> 
        <rosparam param="blue_team_names" subst_value="True">$(arg blue_team_names)</rosparam> 
        <param name="topside_name" value="$(arg topside_name)" />
        <param name="red_team_name" value="$(arg red_team_name)" />
        <rosparam command="load" file="$(arg config_file)" />        
    </node>

</launch>